charts/incubator/st-reborn-server/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: st-reborn-server
+version: 0.0.1
+appVersion: "v1.2.0"
+description: Skyrim Together Reborn server files.
+type: application
+deprecated: false
+home: https://github.com/truecharts/apps/tree/master/charts/incubator/st-reborn-server
+icon: https://truecharts.org/img/chart-icons/st-reborn-server.png
+keywords:
+  - st-reborn-server
+  - gameserver
+sources:
+  - https://github.com/tiltedphoques/TiltedEvolution
+  - https://hub.docker.com/r/tiltedphoques/st-reborn-server
+dependencies:
+  - name: common
+    repository: https://library-charts.truecharts.org
+    version: 10.4.4
+    # condition:
+maintainers:
+  - email: info@truecharts.org
+    name: TrueCharts
+    url: https://truecharts.org
+annotations:
+  truecharts.org/catagories: |
+    - incubator
```

Comment:
Incubator is **NOT** a catagory.
Think "games", "office", "hosting" etc.

Reply:
my bad, ill change it. this ok? 
- games
- hosting